src/clientfactory.ts

```diff
@@ -110,4 +108,12 @@ export class DiscordClientFactory {
             return this.botClient;
         }
     }
+
+    public bindMetricsToChannel(channel: TextChannel) {
+        // tslint:disable-next-line:only-arrow-functions
+        channel.send = function() {
+            MetricPeg.get.remoteCall("channel.send");
+            return channel.send.apply(channel, arguments);
```

Comment:
doesn't this recurse? What if we call `bindMetricsToChannel` twice on a channel object, won't it do the metrics twice?

Reply:
Yes hello I'm dumb, I wonder how we do this sanely.